xc/xenops_server_xen.ml

```diff
@@ -1211,10 +1212,14 @@ module VM = struct
       | HVM hvm_info ->
         let disks = List.filter_map (fun vbd ->
             let id = vbd.Vbd.id in
-            if hvm_info.Vm.qemu_disk_cmdline && (List.mem_assoc id qemu_vbds)
```

Comment:
it would be useful to comment in the commit message why `hvm_info.Vm.qemu_disk_cmdline` is not necessary anymore.